packages/core/admin/admin/src/StrapiApp.js

```diff
@@ -467,5 +471,5 @@ class StrapiApp {
   }
 }
 
-export default ({ adminConfig = {}, appPlugins, library, middlewares, reducers }) =>
-  new StrapiApp({ adminConfig, appPlugins, library, middlewares, reducers });
+export default ({ adminConfig = {}, appPlugins, library, middlewares, reducers, customFields }) =>
```

Comment:
I don't think it makes sense to pass customFields as a param to the StrapiApp. I don't think we need users to be able to customize this right away. We can allow this later if that makes sense